src/main/java/org/craftercms/studio/impl/v2/service/content/ContentTypeServiceImpl.java

```diff
@@ -51,6 +56,7 @@ public ContentTypeServiceImpl(ContentTypeServiceInternal contentTypeServiceInter
      * @throws ServiceLayerException if there is any error finding the items
      */
     @Override
+    @HasPermission(type = DefaultPermission.class, action = PERMISSION_CONTENT_READ)
```

Comment:
Remember to use @HasPermission in combination with @ProtectedResourceId when the permission is site or path dependent. e.g.: https://github.com/craftercms/studio/blob/develop/src/main/java/org/craftercms/studio/impl/v2/service/content/ContentServiceImpl.java#L103

Reply:
Got it. Thanks for the hint!